Update issue templates (#360)
vmware/versatile-data-kit: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve

Reviewer: ```suggestion
about: report an issue and help us improve the project
```

title: ''
labels: bug
assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

**Steps To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version  (please complete the following information):**
 - OS: [e.g. Windows]
 - Version (e.g print output of `vdk version`)

**Additional context**

> Add any other context about the problem here.

Reviewer 2: ```suggestion
Add any other context about the problem here. Consider attaching logs if there are any.
```
